src/main.rs

```diff
@@ -92,6 +93,27 @@ fn film_id_from_response(response: letterboxd::SearchResponse) -> Vec<String> {
         .collect::<Vec<String>>()
 }
 
+/// Get film ids response of list entries request.
+fn film_id_set_from_response(response: letterboxd::ListEntriesResponse) -> HashSet<String> {
+    response
+        .items
+        .into_iter()
+        .map(|entry| entry.film.id)
+        .collect::<HashSet<String>>()
```

Comment:
I think you don't need to specify the type here. It should be inferred from the return type.